comparative_benchmark/jax_xla/run_benchmarks.py

```diff
@@ -117,6 +123,9 @@ def _run_framework_benchmark(
           compile_time_ms,
       "input_data_transfer_ms":
           input_data_transfer_ms,
+      "output_data_transfer_ms":
+          None if not output_data_transfer_latencies else
+          statistics.median(output_data_transfer_latencies),
```

Comment:
GPU benchmarks were completing within 10 mins because almost all JAX workloads were failing. This is the first time it's run all benchmarks successfully so we do need to tune the timeout. The output data transfer latency does add to the total latency but generally not as high as the device latency.